lang/dart.md

```diff
@@ -0,0 +1,206 @@
+---
+lang: Dart
+syntax: dart
+title: Dart is Smart Language!
+author: chandan0000
+---
+
+```dart
+void main(){
+    print("hello world");
+}
+```
+## What is Dart?
+- Dart is a dynamic, class-based, object-oriented programming language with closure and lexical scope. Syntactically, it is quite similar to Java, C, and JavaScript. If you know any of these programming languages, you can easily learn the Dart programming language.
+
+- Dart is an open-source programming language which is widely used to develop the mobile application, modern web-applications, desktop application, and the Internet of Things (IoT) using by Flatter framework. It also supports a few advance concepts such as interfaces, mixins, abstract classes, refield generics, and type interface. It is a compiled language and supports two types of compilation techniques. **[Dart official Website: ](https://dart.dev/tutorials)** https://dart.dev/tutorials
+
+## What is Dart used for?
+Dart programming language can be used for web and mobile applications and is also suitable for server and desktop applications.
+Dart wasn’t popular and was known as an orphan, but it had a compiler for a Dart VM, also Google had all the control about this language.
+
+  **Client Web App Frameworks:**
```

Comment:
Markdown doesn't respect empty spaces at the beginning of the lines. So, it would be more uniform and readable if you can trim extra spaces. Replace line 22 with 👇 

```
**Client Web App Frameworks:**
```